src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4146,6 +4146,65 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/datasets/{id}/snapshotBuilder/concepts/{parentConcept}:
+    get:
+      tags:
+        - datasets
+        - repository
+      description: >
+        Gets all concepts with the specified parent concept.
+      operationId: getConcepts
+      parameters:
+        - in: path
+          description: The UUID of the dataset.
+          name: id
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - in: path
+          description: The id of the parent concept.
+          name: parentConcept
+          required: true
+          schema:
+            type: integer
```

Comment:
Currently concept IDs are always `int` but I wonder if that's something we want as part of our API? It's true in OMOP but isn't a requirement for the lower APIs as far as I can tell.

Reply:
I think `int` is safe for now, because any change we make to this will be expansive rather than reductive - that is, int will need to be included. So we can change this to include more options without making a breaking change